docs/md_files/contributor.md

```diff
@@ -1,32 +1,43 @@
-## About us
-OpenCDA is funded and mainly support by [UCLA Mobility Lab](https://mobility-lab.seas.ucla.edu/).
+## About Us
+OpenCDA is brought to you by [UCLA Mobility Lab](https://mobility-lab.seas.ucla.edu/).
+
+### Supervisor
+Dr. Jiaqi Ma (Associate Professor @ UCLA)
+- Lab Principal Investigator
+- [Linkedin](https://www.linkedin.com/in/jiaqi-ma-17037838/)
+- [Google Scholar](https://scholar.google.com/citations?user=S3cQz1AAAAAJ&hl=en)
+
+### Supervisor
+Dr. Jiaqi Ma (Associate Professor @ UCLA)
+- Project and lab supervisor
+- [Linkedin](https://www.linkedin.com/in/jiaqi-ma-17037838/)
+- [Google Scholar](https://scholar.google.com/citations?user=S3cQz1AAAAAJ&hl=en)
 
 ### Core Developer
 Runsheng Xu (Ph.D. Student @ UCLA):
-- OpenCDA System Architect
+- Project Lead and OpenCDA System Architect
 - [Homepage](https://derrickxunu.github.io/)
 - [Linkedin](https://www.linkedin.com/in/runsheng-xu/)
 
 Xu Han (Ph.D. Student @ UCLA):
-- Map customization
+- Maps
 - [Linkedin](https://www.linkedin.com/in/xu-han-12851a64/)
 
 Hao Xiang (Ph.D. Student @ UCLA):
-- Trajectory prediction module, functionality enhancement
+- Trajectory Prediction, Functionality Enhancement
 - [Homepage](https://xhwind.github.io/)
 - [Linkedin](https://www.linkedin.com/in/hao-xiang-42bb5a1b2/)
 
-### Theoretical Support
 Dr. Xin Xia (Postdoc @ UCLA):
-- Theoretical guidance on cooperative localization
+- Cooperative Localization
 - [Linkedin](https://www.linkedin.com/in/xin-xia-3b6589189/)
 
 Dr. Yi Guo (Postdoc @ UC):
+<<<<<<< HEAD
+- SUMO Integration, platooning algorithm, SUMO Only Mode (under dev)
+- [Linkedin](https://www.linkedin.com/in/yi-guo-4008baaa/)
+=======
 - Help on Sumo part
 - [Linkedin](https://www.linkedin.com/in/yi-guo-4008baaa/)
 
-### Supervisor
-Dr. Jiaqi Ma (Associate Professor @ UCLA)
-- Project and lab supervisor
-- [Linkedin](https://www.linkedin.com/in/jiaqi-ma-17037838/)
-- [Google Scholar](https://scholar.google.com/citations?user=S3cQz1AAAAAJ&hl=en)
+>>>>>>> 232a18fe1724891b03327ab54f14f5849ae11214
```

Comment:
Seems that the merge conflicts are not resolved here.  (like <<<<<HEAD)

Reply:
yes, i fixed it